gothemis/cell/token_protect_test.go

```diff
@@ -359,6 +359,13 @@ func TestTokenProtectDetectExtendedData(t *testing.T) {
 }
 
 func TestTokenProtectDetectCorruptedToken(t *testing.T) {
+	// FIXME(ilammy, 2020-05-25): avoid capacity allocation panics (T1648)
+	// This tests panics on 32-bit architectures due to "int" overflow.
+	// The implementation needs to check for "int" range when casting "C.size_t".
+	if uint64(^uint(0)) == uint64(^uint32(0)) {
```

Comment:
have we started to disable tests because we don't like them? :P

Reply:
Oh, right. For posterity: a fix for this issue is scheduled for 0.13.1. After that decryption of corrupted data of this sort will return an error — likely a different one for 32-bit and 64-bit machines, but still an error.